test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -742,9 +742,9 @@ int run_test_with_multi_import_same_ctx(
     }
     else
     {
-        clVk2CLExternalSemaphore = new clExternalSemaphore(
+        clVk2CLExternalSemaphore = new clExternalImportableSemaphore(
             vkVk2CLSemaphore, context, vkExternalSemaphoreHandleType, deviceId);
```

Comment:
This doesn't necessarily be an importable semaphore. Pretty much all semaphores can be created by OpenCL, imported by Vulkan and can be wait on or signaled from either APIs.

Reply:
It does need to be importable.  It is passed a Vulkan semaphore in the function signature, and is then waited on from OpenCL later.